app/Todo.php

```diff
@@ -14,4 +14,9 @@ class Todo extends Model
     protected $casts = [
         'completed' => 'boolean',
     ];
+
+    public function attributesToArray( $trwet )
+    {
+        $e = 12;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $e. Configured minimum length is 3.](https://www.codacy.com/app/fredlahde/ToDoiy/file/2175950986/issues/source?bid=3163303&fileBranchId=3163382#l20)